src/tile/tile.scss

```diff
@@ -11,309 +11,304 @@
-      width: calc(100% - #{$iui-xl});
-    }
+  ~ .iui-tile-thumbnail-type-indicator,
+  ~ .iui-tile-thumbnail-quick-action {
+    @include iui-blur($opacity: 5);
 
-    .iui-more-options {
-      visibility: visible;
+    > .iui-button-icon {
+      @include themed() {
+        fill: t(iui-color-foreground-accessory);
+        filter: drop-shadow(0 2px 1px rgba(0, 0, 0, t(iui-opacity-5)));
+      }
     }
 
-    .iui-picture {
-      transform: translateZ(0) scale(1.1);
-    }
-  }
+    &:hover,
+    &.iui-active:enabled,
+    &.iui-active:enabled:hover,
+    &.iui-active:active {
+      @include themed() {
+        background-color: rgba(0, 0, 0, t(iui-opacity-4));
+      }
 
-  &.iui-new {
-    .iui-name,
-    .iui-description {
-      font-weight: $iui-font-weight-semibold;
+      .iui-button-icon {
+        @include themed() {
+          fill: t(iui-color-foreground-accessory);
+        }
+      }
     }
   }
+}
 
-  &.iui-selected {
-    @include themed() {
-      outline: solid 2px t(iui-color-foreground-primary);
-      background: linear-gradient(
-        rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6)),
-        rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6))
-      ),
-        linear-gradient(t(iui-color-background-1), t(iui-color-background-1));
-    }
+@mixin iui-tile-thumbnail-type-indicator {
+  @include iui-button-borderless;
+  @include iui-button-size(small, borderless);
+  position: absolute;
+  border-radius: 50%;
+  top: round($iui-baseline * 0.5);
+  left: $iui-s;
+}
 
-    .iui-picture {
-      transform: translateZ(0) scale(1.1);
-    }
-  }
+@mixin iui-tile-thumbnail-quick-action {
+  @include iui-button-borderless;
+  @include iui-button-size(small, borderless);
+  position: absolute;
+  border-radius: 50%;
+  top: round($iui-baseline * 0.5);
+  right: $iui-s;
+}
 
-  &.iui-folder {
-    display: inline-flex;
-    flex-direction: row;
-    flex-wrap: nowrap;
+@mixin iui-tile-thumbnail-badge {
+  position: absolute;
+  bottom: round($iui-baseline * 0.5);
+  width: calc(100% - 2 * #{$iui-sm});
+  height: auto;
+  right: $iui-sm;
+  display: flex;
+  justify-content: flex-end;
+  overflow: hidden;
+
+  > .iui-badge:not(:last-child) {
+    margin-right: $iui-xs;
+  }
+}
 
-    > .iui-thumbnail {
-      height: $iui-baseline * 10;
-      flex: 1;
-      border-bottom: none;
+@mixin iui-tile-buttons {
+  display: flex;
+  flex-shrink: 0;
+  user-select: none;
+  white-space: nowrap;
 
+  > .iui-button {
+    font-size: $iui-font-size;
+    flex: 1;
+    height: auto;
+    padding: $iui-baseline $iui-sm;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    text-align: center;
+    margin: 0;
+    border: initial;
+    border-radius: 0;
+
+    &,
+    &:hover {
       @include themed() {
-        border-right: 1px solid t(iui-color-background-4);
+        border-top: 1px solid t(iui-color-background-4);
+        border-color: t(iui-color-background-4);
```

Comment:
This was the only CSS that I had to add.  Without this line of CSS, the middle dividing line color was incorrect like so:
![Screen Shot 2022-01-26 at 4 02 58 PM](https://user-images.githubusercontent.com/849817/151246568-f86bf84d-158a-48a4-b4c8-7a6dcd90cd97.png)